doc/sphinx/changes.rst

```diff
@@ -384,7 +384,7 @@ Commands and options
   used in the general case. This can create incompatibilities
   in scripts relying on such autogenerated names. The old
   scheme can be reactivated using the deprecated flag
-  :flag:`Program Naming`
+  ``Program Naming``
```

Comment:
So it can no longer be activated right?

Reply:
Indeed. But note that this is the changelog, so it should remain there.

Reply:
Ah whoops.